tests/exception/texception_message_null_byte.nim

```diff
@@ -1,3 +1,11 @@
+discard """
+description: '''
+  . From https://github.com/nim-lang/Nim/issues/13115
```

Comment:
this tests seems it uses a define clause to do things. They look like other test files - but i renamed some of them. 
This test still passes - although it is slow - but his makes me a bit worried that something of this nature can happen in other tests. 
w/e, will cross that bridge when it shows up.

Reply:
Just making sure, but the test calls itself with the define in order to execute various combinations. So long as the code is resilient to renamed that's fine. If you haven't already mentally trace the execution to learn more.

In this case it programmatically gets the file name and it's not hardcoded.